examples/basics/src/layouts/Layout.astro

```diff
@@ -1,50 +1,13 @@
----
-interface Props {
-	title: string;
-}
-
-const { title } = Astro.props;
----
-
 <!doctype html>
 <html lang="en">
 	<head>
 		<meta charset="UTF-8" />
-		<meta name="description" content="Astro description" />
 		<meta name="viewport" content="width=device-width" />
 		<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
 		<meta name="generator" content={Astro.generator} />
-		<title>{title}</title>
+		<title>Document</title>
```

Comment:
nit: Maybe we can use a nicer name like 
```suggestion
		<title>Astro Basics</title>
```

which matches the blog template:
https://github.com/withastro/astro/blob/731b69712b0e9c5d5a4058db3fa3e8bf628c7aad/examples/blog/src/consts.ts#L4

https://github.com/withastro/astro/blob/731b69712b0e9c5d5a4058db3fa3e8bf628c7aad/examples/blog/src/pages/index.astro#L8-L12